docs/src/components/Code.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+
+export const InlineCode = styled.code`
+  padding: 0.25em 0.5em;
+  background: #ddd;
+  border-radius: 0.5em;
+  color: #000;
+  /* monospace typefaces appear a bit larger */
+  font-size: calc(1em - 2px);
+`;
+
+// copied from Prism's theme "Okaidia"
```

Comment:
If you don't mind a different theme, it seems like it'd be easier to offload this onto a package like https://github.com/FormidableLabs/prism-react-renderer

Reply:
Wow, this looks great! 😍 I'll definitely switch over.